### DISCUSSION TITLE
Update from UI?

### DISCUSSION THREAD

**Discussion by deanfourie1** [louislam/dockge#579] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

The ability to update dockge itself from the UI?

### ✔️ Solution

Same as above, just the ability to update from the UI..

### ❓ Alternatives

To connect to host and manually perform a puill.

### 📝 Additional Context

I run dockge to manage my docker environment and this means I do not have to connect to the host to perform pulls.

Yes, to update dockge I have to do just this.

Can we please see the ability to update dockge directly from the UI? Cant be that hard can it?

Thanks!

**Comment by LiranV** [comment]
I just run Dockge with compose as well.
The compose file sits right next to all the others, that way it shows in the Dockge UI and I can update it.
![image](https://github.com/user-attachments/assets/b63e432c-e4ae-423c-8541-81fb2c65c26d)

**Reply by deanfourie1** [reply]
oh so just move the root dockge directory into the stacks?

Haha interesting.

**Reply by mikejaques** [reply]
Putting dockge right in the stacks directory works.   Sure it is not an elegant solution since the web UI spits a big red error instead of stating that dockge is going down and relaunching itself after an update, but it works AFAIK.

EDIT: I was wrong in my comment above.

**Reply by deanfourie1** [reply]
For some reason this does not work for me, after update, the container restarts and does not come back up. I need to login to the host and start it manually.

**Reply by mikejaques** [reply]
actually Dean, I just checked, I thought it did work when I initially tested, but I guess I was wrong as its not working.  So you are correct.  I can throw it in /opt/stacks/, and it'll monitor it, but it won't update and restart itself, it breaks.

Theoretically it would be nice if it could, but currently it can't    :(

**Reply by deanfourie1** [reply]
Yea good to know its not only me then. Would be nice to see this added as an option.
